test/e2e-tests-standing-order.sh

```diff
@@ -3,8 +3,28 @@ set -e
 
 cd dex-contracts/
 
+truffle exec scripts/sell_order.js 1 2 1 1 1
+
 # Place standing order in current Auction (accountId, buyToken, sellToken, minBuy, maxSell)
-truffle exec scripts/standing_order.js 0 1 2 12 12
+truffle exec scripts/standing_order.js 0 1 2 1 1
```

Comment:
If we are using the first 50 account slots for reserved accounts, does this mean we have adjusted all the regular orders in the other E2E tests?

I see the sell order directly above is referring to accountId = 1

Reply:
Reserved accounts can place normal sell orders as well, so I don't think it should affect the e2e tests.

Reply:
Ahhh yes I see. I suppose I hadn't noticed that there is not yet an implementation for account reservation.